### DISCUSSION TITLE
How to use CSS-in-JS library stitches?

### DISCUSSION THREAD

**Discussion by gutenye** [resend/react-email#568] [Q&A]
Hey guys

I follow the guide [here](https://stitches.dev/docs/server-side-rendering), you need call `getCssText()` after `renderToStaticMarkup()`. Is there a hook available in `email export` command something like this?

```tsx
const EmalTemplate = () => (
  <Html>
    <Head>
       <style id="stitches"></style>
    </Head>
  </Html>
)
const html = renderToStaticMarkup(<EmailTemplate />)
const style = getCssText()
html.replace('<style id="stitches"></style>', `<style id="stitches">${style}</style>`)
```

**Comment by bukinoshita** [comment]
We don't have at the moment, just to clarify the code you shared it's working?

**Reply by gutenye** [reply]
I end up using inline CSS.
